appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -17,16 +18,18 @@ else{
 }
 
 function Rebase-Uri($originalURI){
```

Comment:
Naming: For consistency with other variables in  the code, and with .NET coding conventions (where acronyms start with a single capital letter), rename this to `$originalUri`.